src/docs/Updates/Roadmap/index.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Roadmap'
+order: 3
+description: 'Discover what we are working on right now, and how you could contribute to Mozaic'
+---
+
+## Q1 2020 (from 01/01/2020 to 01/04/2020)
+
+#### Create a bêta version of the Mozaic-vue library
+
+- Implement CI / CD and code quality tools
+- Create contributing and coding guidelines
+- Create and migrate at least all the forms components existing in Mozaic into Mozaic-vue
+- Link Mozaic-vue in the Mozaic website
+- Support contribution from other teams
+
+#### Core team components migration into Mozaic
+
+- Help the core team migrate existing components into mozaic
+  - popin / modal component
+  - visibility utilities (for responive and screen readers)
+  - reset / a css file to reset and define basic global styles
+  - images / videos components
+  - separators / dividers component
+  - wysiwyg scope styles (styles required for wysiwyg output)
+
+#### Support Core team into migrating Mozaic into common socle
+
+- Implement mozaic components as freemarker templates
+- Distribute larger parts of Mozaic through common socle
+
+#### Improve Mozaic UI/UX and information architecture
+
+- Redesign home page
+- Improve User interface for a more polished look
+- Improve documentation about contributting
```

Comment:
Since this is one of the team's "top priorities" and we're working on it now, can you pull it up right after the **"Create a bêta version of the Mozaic-vue library"** section?

Reply:
- Improve User interface for a more polished look 
- Improve the global ergonomy (table of content, search box, etc...)

Reply:
~~Improve documentation about contributting~~ <= We can remove this in this section and create a specific one dedicated to the contribution.

Something like:

#### Make contributing easy
- Organising contributions
- Improve documentation about contributting
- Create a contributing funnel
- Reduce PR and merge time as much as possible
- Set up a voting system for missing patterns
- Following and supporting contributors to improve Mozaic